### ISSUE TITLE
Lack of keyboard navigation in quick-search

### ISSUE THREAD

**Issue by vijha742** [web-dev-Vivek/Portfolio_vikas#2]
<img width="715" height="546" alt="Image" src="https://github.com/user-attachments/assets/67653bbd-c1e7-487f-86b1-1ad88b1eb54e" />
Like there is no up-down arrow navigation to access the elements listed, which defeats the entire purpose of search bar.
